website/docs/components/copy/snippet/partials/code/component-api.md

```diff
@@ -0,0 +1,22 @@
+## Component API
+
+This component uses [ember-cli-clipboard](https://github.com/jkusa/ember-cli-clipboard) under the hood.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="color" @type="enum" @values={{array "secondary" "tertiary" }} @default="tertiary"/>
+  <C.Property @name="isFullWidth" @type="boolean" @default="false">
+    Indicates that the component should take up the full width of the parent container.
+  </C.Property>
+  <C.Property @name="textToCopy" @type="string" @required="true">
+    String value or action that returns a string to be copied.
+  </C.Property>
+  <C.Property @name="onSuccess" @type="action">
+     Action sent on a successful copy.
+  </C.Property>
+    <C.Property @name="onError" @type="action">
+     Action sent on a failed copy.
+  </C.Property>
```

Comment:
The `onSuccess` / `onError` methods are not accessible to the consumer, are internal

```suggestion
```